### PR TITLE
Fix problem with multiple ? in urls

### PR DESCRIPTION
**Pull request by jjmurre**
When using url params like _format=json, the embedded
temporal urls were having  more than one '?' in the url, leading to invalid url.

# This Pull request contains changes to:

Fix invalid temporal url in the "_links" section of the HAL output.

# In case changes new features added

- [ ] Customer friendly documentation added into `docs/`.
- [ ] Developer friendly documentation added into `DEVELOPMENT|README.md`

# In case breaking changes introduced into API

- [ ] There is customer notification plan: ....

# In case this PR reverts changes in repo

- [ ] Extraq details describing reasoning: ...

### CODE REVIEW

File: src/tests/test_dynamic_api/test_temporal_actions.py [hunk 1]
```diff
@@ -180,3 +180,16 @@ def test_serializer_temporal_request_corrects_link_to_temporal(
...
+    ):
+        """Prove that extra url parameters do not interfere with the existing
+        url parameters for temporality."""
+        url = reverse("dynamic_api:gebieden-stadsdelen-list")
+        response = api_client.get(f"{url}?_format=json")
+        data = read_response_json(response)
+
+        # Check if there is only one '?' in the temporal urls
+        assert len(data["_embedded"]["stadsdelen"][1]["_links"]["self"]["href"].split("?")) == 2
+        assert len(data["_embedded"]["stadsdelen"][1]["_links"]["wijk"][0]["href"].split("?")) == 2
```
Comment (vdboor) [inline src/tests/test_dynamic_api/test_temporal_actions.py:195]: waarom niet de parameters zelf checken? :-)   `"?_format=json&..." in ...` bijvoorbeeld.

Comment (jjmurre) [reply src/tests/test_dynamic_api/test_temporal_actions.py:195]: Ja, wellicht nog wat duidelijker. Alhoewel, je er niet van uit kunt gaan dat de volgorde van de parameters vastligt, maar daar kun je rekening mee houden.

Comment (jjmurre) [reply src/tests/test_dynamic_api/test_temporal_actions.py:195]: Ik doet wel iets obv. urlparse, dat is nog wat explicieter.

**Review by ngaranko — APPROVED** [review]
LGTM

**Pull request merged**